packages/ibm-products/src/components/Datagrid/Datagrid/addons/Filtering/hooks/useFilters.js

```diff
@@ -176,6 +179,53 @@ const useFilters = ({
       );
     }
 
+    const renderCheckboxes = () => {
+      if (variation === PANEL && filtersState[column].value.length > 10) {
+        return (
+          <OverflowCheckboxes
+            components={components}
+            type={type}
+            column={column}
+            setFiltersState={setFiltersState}
+            filtersState={filtersState}
+            applyFilters={applyFilters}
+          />
+        );
+      }
+
+      return (
+        <FormGroup {...components.FormGroup}>
+          {filtersState[column].value.map((option) => (
+            <Checkbox
+              key={option.id}
+              {...option}
+              onChange={(isSelected) => {
+                const checkboxCopy = filtersState[column].value;
```

Comment:
The contents of this `onChange` look to be the exact same as the `onChange` in the [new component](https://github.com/carbon-design-system/ibm-products/pull/3560/files#diff-6ec7ccb1e61376029671970528fcd395601380c7d16af759baa69ba992d7c9a6R27-R44), `OverflowCheckboxes.js`. Can we put this in a utility that can be used from both checkboxes?

Reply:
Good call